shared/chat/inbox/row/add-new-row.js

```diff
@@ -0,0 +1,97 @@
+// @flow
+import React from 'react'
+import {Text, Icon, Box, ClickableBox, LoadingLine} from '../../../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../../../styles'
+import {isMobile} from '../../../constants/platform'
+import {branch} from 'recompose'
+
+let KeyHandler
+if (!isMobile) {
+  KeyHandler = require('../../../util/key-handler.desktop').default
+}
+
+const _AddNewRow = ({
+  onNewChat,
+  onJumpToChat,
+  isLoading,
+}: {
+  onNewChat: () => void,
+  onJumpToChat: () => void,
+  isLoading: boolean,
+}) => (
+  <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', minHeight: 48, position: 'relative'}}>
+    <ClickableBox
+      style={{
+        ...globalStyles.flexBoxRow,
+        alignItems: 'center',
+        flexGrow: 1,
+        flexShrink: 0,
+        justifyContent: 'center',
+        paddingLeft: globalMargins.medium,
+        paddingRight: globalMargins.medium,
+      }}
+      onClick={onNewChat}
+    >
+      <Icon type="iconfont-new" style={{color: globalColors.blue, marginRight: 9}} />
+      <Text type="BodyBigLink">New chat</Text>
+    </ClickableBox>
+    {isLoading &&
+      <Box style={{bottom: 0, left: 0, position: 'absolute', right: 0}}>
+        <LoadingLine />
+      </Box>}
+  </Box>
+)
+
+// const _AddNewRow= ({
```

Comment:
originaly iwas going to do the filtering, but then i split that out so fred could branch off of this so he'll actually be uncommenting this and deleting the other stuff